src/components/erc20/erc20_app.tsx

```diff
@@ -1,22 +1,56 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import { Route, Switch } from 'react-router';
+import { ThemeProvider } from 'styled-components';
 
 import { ERC20_APP_BASE_PATH } from '../../common/constants';
 import { AdBlockDetector } from '../../components/common/adblock_detector';
-import { GeneralLayoutContainer } from '../../components/general_layout';
+import { GeneralLayout } from '../../components/general_layout';
+import { setThemeColor } from '../../store/ui/actions';
+import { Theme } from '../../themes/commons';
+import { getThemeByRoute } from '../../themes/theme_meta_data_utils';
 
 import { ToolbarContentContainer } from './common/toolbar_content';
 import { Marketplace } from './pages/marketplace';
 import { MyWallet } from './pages/my_wallet';
 
+interface StateProps {
+    theme: Theme;
+}
+
+interface DispatchProps {
+    setThemeColor: (theme: Theme) => any;
+}
+
+type Props = StateProps & DispatchProps;
+
 const toolbar = <ToolbarContentContainer />;
 
-export const Erc20App = () => (
-    <GeneralLayoutContainer toolbar={toolbar}>
-        <AdBlockDetector />
-        <Switch>
-            <Route exact={true} path={`${ERC20_APP_BASE_PATH}/`} component={Marketplace} />
-            <Route exact={true} path={`${ERC20_APP_BASE_PATH}/my-wallet`} component={MyWallet} />
-        </Switch>
-    </GeneralLayoutContainer>
-);
+const Erc20AppContainer = (props: Props) => {
+    const themeColor = getThemeByRoute(ERC20_APP_BASE_PATH);
+    props.setThemeColor(themeColor);
+    return (
+        <ThemeProvider theme={themeColor}>
+            <GeneralLayout toolbar={toolbar}>
+                <AdBlockDetector />
+                <Switch>
+                    <Route exact={true} path={`${ERC20_APP_BASE_PATH}/`} component={Marketplace} />
+                    <Route exact={true} path={`${ERC20_APP_BASE_PATH}/my-wallet`} component={MyWallet} />
+                </Switch>
+            </GeneralLayout>
+        </ThemeProvider>
+    );
+};
+
+const mapDispatchToProps = (dispatch: any) => {
+    return {
+        setThemeColor: (theme: Theme) => dispatch(setThemeColor(theme)),
```

Comment:
I think that now, having the theme on the store does not make sense any more, we could just remove it, or split it into two (one for erc20 and erc720). But It won't be used any more, so I think it's better to just remove it. Thoughs? @mariano-aguero @unjapones @fvictorio

Reply:
Mmmm I think it is not necessary, split it? neither, the theme is for the application(no matter what market you're ) and I think It should not be by market. 
Regards

Reply:
As discussed internally, we decided to remove the themes from the store and two have two themes for the app.